app/controls/actionlabel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "actionlabel.h"
+
+#include <QAction>
+
+ActionLabel::ActionLabel(QWidget *parent) :
+	ActiveLabel(parent),
```

Comment:
Thanks for pointing this out, looks like my LInux and Windows dev machines have different formatting settings in QtCreator — I'll fix it.